pkg/cmd/codespace/select_test.go

```diff
@@ -0,0 +1,95 @@
+package codespace
+
+import (
+	"context"
+	"errors"
+	"os"
+	"fmt"
+	"testing"
+
+	"github.com/cli/cli/v2/internal/codespaces/api"
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+const CODESPACE_NAME = "monalisa-cli-cli-abcdef"
+const OUTPUT_FILE_PATH = "codespace-selection-test.log"
```

Comment:
Why don't you generate this into whatever temp/tmp storage on the platform you are on? Saves you having to special case it in the local git tree with gitignore.  Or can you just ensure the file gets cleaned up at the end of the test run.

Reply:
https://pkg.go.dev/io/ioutil#TempFile ☝️

Reply:
Used `TempFile` instead 👍